src/main/java/com/fasterxml/aalto/sax/SAXParserFactoryImpl.java

```diff
@@ -95,7 +98,8 @@ public void setFeature(String name, boolean enabled)
 
             switch (stdFeat) {
             case EXTERNAL_GENERAL_ENTITIES:
-                ok = !enabled;
```

Comment:
As per my notes elsewhere, no. This is NOT what "EXTERNAL_GENERAL_ENTITIES" in general would mean.

Should instead use either `P_RETAIN_ATTRIBUTE_GENERAL_ENTITIES` id, or (if we want it for some reason), another SAX-and-Aalto-specific name.